mteb/tasks/STS/por/SickBrSTS.py

```diff
@@ -0,0 +1,51 @@
+from __future__ import annotations
+
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+from ....abstasks.AbsTaskSTS import AbsTaskSTS
+
+
+class SickBrSTS(AbsTaskSTS):
+    metadata = TaskMetadata(
+        name="SICK-BR",
+        dataset={
+            "path": "eduagarcia/sick-br",
+            "revision": "0cdfb1d51ef339011c067688a3b75b82f927c097",
+        },
+        description="SICK-BR is a Portuguese inference corpus, aligned to and translated from SICK",
```

Comment:
I have a feeling we are not that fond of having translated datasets (right, @KennethEnevoldsen ?), is there a particular reason this dataset should make an exception.

Reply:
@x-tabdeveloping sorry, I did not realize that was the case.

Feel free to close this PR then.

Reply:
That said, you should remove the SickrPLSTS (https://github.com/embeddings-benchmark/mteb/blob/main/mteb/tasks/STS/pol/PolishSTS.py#L7) as well then -- here is what it's paper (https://aclanthology.org/2020.lrec-1.207.pdf) says:

> We share a manually translated version of SICK (Sentences Involving Compositional Knowledge) dataset,
containing 10,000 pairs of sentences annotated with
textual entailment labels and semantic relatedness
scores.

Reply:
@KennethEnevoldsen I would greatly appreciate if you could share your thoughts here so that I can either finish this up or close it altogether?

Thanks!

Reply:
Understood, thanks @KennethEnevoldsen. I believe this particular dataset was only bootstrapped from English translation -- it was then manually checked, even twice in some cases.

This is from Section 3.2 of the associated paper (https://linux.ime.usp.br/~thalen/SICK_PT.pdf):

> The 6076 unique sentences of the corpus were equally distributed among the
10 annotators. We used an online platform for the checking. This made it possible
for annotators to look at each others’ work when translating their sentences. We
also kept an online forum for discussing issues, where more than 2k messages were
exchanged during this work. The glossary was always updated when a solution
was reached. Each annotator could also mark out complex sentences that they
thought needed further review. Differently from other corpora creation processes
we know about, our annotators could always say they were not able to annotate
something, an easy strategy that helps to ensure the quality of the work. Finally,
an experienced annotator double checked all sentences considered complex and
proposed a final translation for these sentences.

Would you find that to be enough for the dataset to be considered?

Reply:
Ahh, so it is human-translated. I think this is a fine dataset to add.

Reply:
I would also make this clear in the description.

Reply:
Understood, thank you @KennethEnevoldsen, will certainly make it so!